### PR TITLE
Update metadata and do minor fixes

### PR DESCRIPTION
**Pull request by manuranga**

### CODE REVIEW

File: point-to-point-channel/point-to-point-channel.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-tagline: 
-desc: 
-category: Message Channel
+name: Point-to-Point Channel
```
Comment (LakshanWeerasinghe) [inline point-to-point-channel/point-to-point-channel.yml:1]: we don't have a name field in other yml files.

File: publish-subscribe-channel/publish-subscribe-channel.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+category: Messaging Channels
+index: 7
+link: https://www.enterpriseintegrationpatterns.com/patterns/messaging/PublishSubscribeChannel.html
```
Comment (LakshanWeerasinghe) [inline publish-subscribe-channel/publish-subscribe-channel.yml]:
```suggestion
link: https://www.enterpriseintegrationpatterns.com/patterns/messaging/PublishSubscribeChannel.html

```

**Review by LakshanWeerasinghe — COMMENTED** [review]
Need to remove this `node_modules/node_modules.yml` empty file as well.

File: point-to-point-channel/point-to-point-channel.yml (hunk 1 above)
Comment (manuranga) [reply point-to-point-channel/point-to-point-channel.yml:1]: only needed if name has `-`s.

**Review by LakshanWeerasinghe — APPROVED** [review]

**Pull request merged**